plugins/rplanners/ParabolicPathSmooth/ppramp.cpp

```diff
@@ -254,7 +254,7 @@ bool PPRamp::SolveFixedTime(Real amax,Real endTime)
 
 bool PPRamp::SolveMinAccel(Real endTime)
 {
-    Real switch1,switch2;
+    Real switch1 = 0.0,switch2 = 0.0;
```

Comment:
@Puttichai apparently, `CalcMinAccel(endTime,1.0,switch1)` has code paths that do not set `switch1` to any value. Shall I use 0.0 or -1.0, as we did for `CalcSwitchTimes()` ?

Reply:
done